sass/components/_mentions.scss

```diff
@@ -73,6 +72,12 @@
         }
     }
 
+    // .mentions__name is way too multi-purpose. This rule targets its use in
+    // SwitchChannelSuggestion, applying margin in a way that handles null elements correctly.
+    .status, .mentions__fullname {
```

Comment:
`.status` and `.mentions__fullname` is only used in `SwitchChannelSuggestion`. I'd love to see us holistically tackle the CSS here to avoid "weird" edge cases like this.

Reply:
Yeah, the autocomplete is another component in desperate need of a rewrite